code/modules/projectiles/ammo_datums.dm

```diff
@@ -1443,15 +1443,25 @@
 //unused and not working. need to refactor MD code. Unobtainable.
 //intended mechanic is to have xenos hit with it show up very frequently on any MDs around
 /datum/ammo/bullet/lever_action/tracker
-	name = "tracking lever-action bullet"
+	name = ".458 tracker round"
 	icon_state = "redbullet"
 	damage = 70
 	penetration = ARMOR_PENETRATION_TIER_3
 	accuracy = HIT_ACCURACY_TIER_1
 	handful_state = "tracking_lever_action_bullet"
 
+/datum/ammo/bullet/lever_action/tracker/proc/tracker_remove(mob/M, icon)
+	REMOVE_TRAIT(M, TRAIT_SPOTTER_LAZED, TRAIT_SOURCE_EQUIPMENT(2000))
```

Comment:
What's 2000 here?

Reply:
It's the CAS tracker for the bullet.

The way I see it I had 4 solutions for CAS ID for this PR:
1: Give every single bullet a CAS ID round-start which would increment the CAS ID number by +40
2: Give a incremented CAS ID every time a tracking bullet fires
3: Set the CAS ID to some really high number that I don't think can be reached in-game like 2000
4: Rework the gun systems to support having a CAS ID linked to the gun itself for my one-off item.

I chose number 3 so I set the CAS ID to 2000 because I don't think we'll ever get to 2000 laser binos and rangefinders on the map.